Example/.swiftlint.yml

```diff
@@ -46,9 +46,9 @@ excluded:
   - Carthage
   - ./*/SnapshotHelper.swift
 
-# # Include the development pod
-# included:
-#   - ../Sources/Classes
+# Include the development pod
```

Comment:
@xavierLowmiller This will be overridden when there are new coding-guidelines swiftlint rules. But as `.swiftlint.yml` is not gitignored, the developer can merge the changes correctly (maybe). Sadly it is not possible to include the Sources from a `.swiftlint.yml` file from a subfolder.